demoapp/package.json

```diff
@@ -19,15 +19,14 @@
     }
   },
   "dependencies": {
-    "cordova-android": "^7.0.0",
-    "cordova-ios": "^5.0.0",
```

Comment:
Is removing `cordova-ios` dependency intentional? @russelarms

Reply:
Fixed